requirements.txt

```diff
@@ -1,4 +1,4 @@
 Sphinx>=3.0
```

Comment:
Sphinx 3 may be, but Sphinx 4 is not—it was only released 9 months ago, in May 2021—and unfortunately, a number of popular Sphinx plugins (e.g. Sphinx-Multiversion) and themes still don't fully support it. I'm not sure I see a compelling reason to require such a new version unless this repo is using Sphinx 4-only syntax, which for a relatively small and simple project like this is not likely the case.

Reply:
This project is already using Sphinx 4, because it depends on Furo which requires Sphinx 4+. The most recent build used 4.4.0.

I also find the point that extensions not used in this documentation don’t support Sphinx 4 to be odd — it doesn’t matter? There’s always going to be bitrot in the broader ecosystem. Just because there exists an extension that hasn’t migrated doesn’t mean that projects that don’t use it, can’t move forward.

Reply:
I don't see how the `>=3.0` would prevent anyone from moving forward?
I'd drop the constraint entirely rather than tighten it. But it really doesn't matter, IMO.

Reply:
If Furo requires Sphinx 4 (which I wasn't aware of) then the correct constraint should indeed be Sphinx>=4.0 as @AA-Turner suggests. Otherwise, the build will likely fail for users who install the requirements in an existing environment with Sphinx >4.0, and it sends a potentially confusing and misleading signal about the project's Sphinx support. My point was only that we shouldn't add a high arbitrary constraint without a reason behind it (which indeed there is).

Reply:
> Otherwise, the build will likely fail for users who install the requirements in an existing environment with Sphinx >4.0

Pip's install logic for the past ~2 years has used a proper resolver, so as long as `furo` declares version constraints on Sphinx properly (I haven't checked, but I trust it does), the effective constraint is whatever furo says rather than the `>=3.0` listed. 

#26 removes the constraint to stop providing a false signal on compatability.

A

Reply:
Ah, it hadn't occurred to me that `furo` was itself deployed as a Python package (and would thus of course be able to appropriately constrain dependency versions). In that case, removing the constraint and letting it be dictated by the appropriate dependencies would indeed make the most sense, as #26 proposes.